randovania/games/prime1/json_data/Phazon Mines.txt

```diff
@@ -1280,15 +1281,29 @@ Extra - aabb: [122.126015, -44.231895, -102.18276, 201.19897, 105.61425, -30.731
   * Extra - nonstandard: False
   > Door to Dynamo Access
       All of the following:
+          After Invisible Drone
           Any of the following:
               # Fast climb with Standable: https://youtu.be/MtpgWKgaF-U
               Space Jump Boots
               Morph Ball Bomb and Morph Ball and Complex Bomb Jump (Expert) and Standable Terrain (Intermediate)
           Combat (Intermediate) or Normal Damage ≥ 125 or Shoot Ice Beam
   > Door to Quarantine Access A
       All of the following:
-          Morph Ball and Power Bomb and After Invisible Drone
-          Combat (Intermediate) or Normal Damage ≥ 200 or Shoot Ice Beam
+          Morph Ball and Power Bomb
+          Any of the following:
+              All of the following:
+                  After Invisible Drone
+                  Ice Beam or Combat (Intermediate) or Normal Damage ≥ 200
+              Before Invisible Drone and Knowledge (Beginner)
+  > Event - Security Drone
```

Comment:
Please find a way to add the logic you desire without copy-pasting logic like this. Feel free to add new nodes like `Before Fight` or `Room Center` for example

Reply:
I will handle this today